docs/contributing/coding-style.md

```diff
@@ -1018,6 +1079,166 @@ Still, in some cases, you may need to change the generated code to meet your req
            // generated test code
    ```
 
+1. Context
+
+   If the target function accepts context as an input argument, the test code generated will include it in `args`.
+
+   e.g. 
+
+   ```go
+    type args struct {
+        ctx context.Context
+    }
+    ...
+   ```
+
+   But in test implementation, it is hard to manage the context lifecycle. We want to guarantee the context is closed after the test is executed, to avoid any missing termination of the process.
```

Comment:
```suggestion
   But in test implementation, it is hard to manage the context lifecycle.
   We want to guarantee the context is closed after each test is executed to avoid any missing termination of the process.
```